continuous_integration/recipes/distributed/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
 python:
     - 3.8
     - 3.9
+build_ext:
+    - "cython"  # [linux]
+    - "python"
```

Comment:
```suggestion
cython_enabled:
    - True   # [linux]
    - False
```

Reply:
Yeah I like this handling, will commit these suggestions